### PR TITLE
enable pipeline to run quantization tests

### PR DESCRIPTION
**Pull request by yufenglee**
**Description**: Describe your changes.
set up quantization tests

### CODE REVIEW

File: onnxruntime/test/python/quantization/calibrate_test.py [hunk 1]
```diff
@@ -250,10 +250,11 @@ def test_quant_param_calculation(self):
         #test calculation of quantization params
         #TO_DO: check rmin/rmax
         dict_for_quantization = calibrater.get_intermediate_outputs()
+        print(len(dict_for_quantization))
```
Comment (tracysh) [inline onnxruntime/test/python/quantization/calibrate_test.py:253]: stray debugging change?

**Review by zhanghuanrong — DISMISSED** [review]

**Review by zhanghuanrong — APPROVED** [review]

**Pull request merged**